docs/docs/configuration.md

```diff
@@ -77,6 +77,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.metadata.previous-versions-max                 | 100                         | The max number of previous version metadata files to keep before deleting after commit                                                                                                            |
 | write.spark.fanout.enabled                           | false                       | Enables the fanout writer in Spark that does not require data to be clustered; uses more memory                                                                                                   |
 | write.object-storage.enabled                         | false                       | Enables the object storage location provider that adds a hash component to file paths                                                                                                             |
+| write.object-storage.omit-partition-values           | false                       | Excludes the partition values from the file path                                                                                                                                                  |
```

Comment:
This property introduces a "double-negative".  I think we should invert this to be something like `write.object-storage.partitioned-paths = true`

Reply:
Updated